android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.java

```diff
@@ -170,14 +173,16 @@ private void updateKernelRNOkHttp() {
   }
 
   // Don't call this until a loading screen is up, since it has to do some work on the main thread.
-  public void startJSKernel(Activity activity) {
+  public void startJSKernel(AppCompatActivity activity) {
     if (Constants.isStandaloneApp()) {
       return;
     }
     setActivityContext(activity);
 
     SoLoader.init(mContext, false);
 
+    ExperienceActivityUtils.overrideUserInterfaceStyle(mExponentManifest.getKernelManifest(), activity);
```

Comment:
cc @esamelson I need some expertise from you in this thing 😉 I've added `ExperienceActivityUtils.overrideUserInterfaceStyle` that works like a charm in `ExperienceActivity`, but I also need to propagate it's behaviour to `HomeActivity`. Because I couldn't find any path to obtain `kernelManifest` (that describes specified behaviour - `automatic`) I found this place in codebase that has references to both. I'm not too proud of this though. 😓

Reply:
When I thing about it I can simply hardcode on native level that `Expo Client` should provide behaviour according to `automatic` option, but (to be honest) I don't like this idea 😛 (I prefer treating `Expo Home` as another `Experience` with a little different logic).

Reply:
`ExponentManifest` is actually a singleton at runtime so you should be able to call into it directly from `HomeActivity` to get the kernel manifest. We use this funky inject thing for some of the singletons so you'll need to add https://github.com/expo/expo/blob/97671b9852f4fcc9f449a0b231687502024f7049/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java#L111-L112 and https://github.com/expo/expo/blob/97671b9852f4fcc9f449a0b231687502024f7049/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java#L159 to HomeActivity (replacing `ExperienceActivity.class` with `HomeActivity.class`) to get access to the singleton, but that should work. Does that make things better? 😄

Reply:
what are we doing for orientation in `HomeActivity` currently, is that hardcoded somewhere? if so, perhaps that's just the safest thing to do here too and we can create an issue to follow up with to come up with a better solution for both orientation and color scheme in `HomeActivity`

Reply:
As I'm not this familiar with `ScreenOrientation` (I only found some reference in `AndroidManifest.xml`) I went with `@Inject` method 😉